### PR TITLE
Update Venue to be queryable from REST/GraphQL api

### PR DESCRIPTION
**Pull request by vvasiloud**
Hello,

Currently I am using the-events-calendar with external plugins to add graphql support in the api.
The post type  `Venue`  seems not be publicly exposed (without authentication), the following change will update the visibility to public property from the rest/graphql api.

External plugins used:
- WP GraphQL 1.6.7
- QL Events 0.0.1 (adding the-events-calendar to graphql properties)

Example query reproducing the error:

`query getEvents() {
  events {
    nodes {
      id
      title
      venue {
        title
        address
        city
        country
        slug
      }
    }
  }
}
`

### CODE REVIEW

**Comment by Camwyn** [comment]
Hey @vvasiloud

Thanks for this!

This can be accomplished currently via a filter: [tribe_events_register_venue_type_args](https://github.com/the-events-calendar/the-events-calendar/blob/master/src/Tribe/Venue.php#L152)

I'm going to look into why it's set to false - and if we change this, whether we should also consider doing the same for Organizers.

**Review by lucatume — COMMENTED** [review]
I do not think this is the place to make the Venue post type public, open to discussion.

File: src/Tribe/Venue.php [hunk 1]
```diff
@@ -8,7 +8,7 @@ class Tribe__Events__Venue extends Tribe__Events__Linked_Posts__Base {
 	 * @var array
 	 */
 	public $post_type_args = [
-		'public'              => false,
+		'public'              => true,
```
[the remark this answers is not in the document]
Comment (Camwyn) [reply src/Tribe/Venue.php:11]: Yeah, I did explore some and I think I agree that it should stay non-public.
**but** you can still use the filter to change it for a specific use case.

@vvasiloud can you provide us with a specific use case/argument for making it public?

[the remark this answers is not in the document]
Comment (vvasiloud) [reply src/Tribe/Venue.php:11]: @Camwyn Basically I use this field to show a list of events in graphql with a section about where the venue is (venue name, address etc.)

Similarly to the bottom of the demo page here: https://demo.theeventscalendar.com/event/event-single/

For the moment I have used a filter to override it but I was wondering why is not exposed by default if is already shown in the public plugin page.

Thanks.

**Comment by Camwyn** [comment]
Closing this as we will not move forward with making this public by default

**Pull request closed without merging**